database/migrations/2016_07_27_172344_create_kudos_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->integer('taxonomy_id')->index()->comment('The taxonomy_term.tid that this kudos belongs to.');
             $table->timestamps();
 
-            $table->unique(['file_id', 'drupal_id', 'northstar_id', 'taxonomy_id']);
+            $table->unique(['file_id', 'drupal_id', 'northstar_id', 'taxonomy_id'], 'file_drupal_ns_taxonomy_unique');
```

Comment:
Is this just so that index isn't named something with `id` in it 4 times?

Reply:
Yeah sorry, should probably have added comments for these. By default, Laravel makes an index with the combined column names `file_id_drupal_id_northstar_id_taxonomy_id_unique` and that was giving an "index name too long" error.

Reply:
I approve of that error that is too long! makes sense!